lib/logflare_web/live/search_live/logs_search_lv.ex

```diff
@@ -887,4 +891,31 @@ defmodule LogflareWeb.Source.SearchLV do
     |> assign(:querystring, qs)
     |> assign(:lql_rules, lql_rules)
   end
+
+  defp check_suggested_fields(_lql_rules, _source, %{assigns: %{force_query: true}} = socket) do
+    {:ok, assign(socket, :force_query, false)}
+  end
+
+  defp check_suggested_fields(lql_rules, %{suggested_fields: ""}, socket) do
+    {:ok, socket}
+  end
+
+  defp check_suggested_fields(
+         lql_rules,
+         %{suggested_fields: suggested_fields},
+         %{assigns: %{force_query: false}} = socket
+       ) do
+    suggested_present =
+      suggested_fields
+      |> String.split(",")
+      |> Enum.map(fn
+        "m." <> suggested_field -> "metadata." <> suggested_field
```

Comment:
Don't think this is needed

Reply:
unfortunately we need it, if a user sets the keys as they use them (e.g. `m.project`) then it will not work since the LQL parser converts the `m` to `metadata`